### PR TITLE
[exclusive-accordion] Add sections on existing details issues and spicy sections

### PR DESCRIPTION
**Pull request by dbaron**

### CODE REVIEW

**Comment by dbaron** [comment]
See [preview](https://deploy-preview-746--open-ui.netlify.app/components/accordion.explainer/).

File: site/src/pages/components/accordion.explainer.mdx [hunk 1]
```diff
@@ -289,3 +317,21 @@ while intersecting,
 are different enough
```
Comment (mfreed7) [inline site/src/pages/components/accordion.explainer.mdx:317]: Is it worth adding a link to https://github.com/openui/open-ui/issues/559 above?

I also don't quite understand the difference between "panelset" and "spicy sections". They both seem like a thing that goes between accordions and tabs. I'm not saying you need to change anything about the explainer, per se, just asking if you agree. If so, I suppose you could add a sentence along those lines. Or combine the two sections together. Or leave them as-is.

**Review by mfreed7 — APPROVED** [review]
LGTM, just one rambling comment. Take it or leave it.

**Comment by bkardell** [comment]
Yeah, sorry I didn't have enough time to comment on this but spicy-sections _is_ panel-set (or you might see panselset).  I made a comment in the commit.

**Comment by bkardell** [comment]
probably close enough, yeah. @mirisuzanne wdyt?

**Comment by mirisuzanne** [comment]
Yeah, that makes sense to me.

**Pull request merged**